services/wireguard/wireguard.go

```diff
@@ -58,7 +58,7 @@ func (method Payment) GetPrice() money.Money {
 type ConnectionEndpoint interface {
 	Start(config *ServiceConfig) error
 	AddPeer(publicKey string, endpoint *net.UDPAddr) error
-	PeerStats() (consumer.SessionStatistics, error)
+	PeerStats() (stats consumer.SessionStatistics, lastHandshake int, err error)
```

Comment:
hm, I thought that last handshake time is a part of the peer stats. Maybe it should go to the `SessionStatistics` or another struct, but it's still stats.

Reply:
Yeap. So instead of returning consumer.SessionStatitics which I bet is something defined as common structure, we need to return wg specific stats with last handshake included.

Reply:
Replaced by the single struct.